### ISSUE TITLE
Street view

### ISSUE THREAD

**Issue by msreekm** [fullstackreact/google-maps-react#270]
Is it possible to load google-maps-react with streetview as the default ? any examples?

**Comment by Qwiso** [comment]
No. This repo is a minimal implementation of the Maps Javascript API

Your only options would be:
roadmap displays the default road map view. This is the default map type.
satellite displays Google Earth satellite images.
hybrid displays a mixture of normal and satellite views.
terrain displays a physical map based on terrain information.

https://developers.google.com/maps/documentation/javascript/maptypes#BasicMapTypes

**Comment by Qwiso** [comment]
There are no examples but I guess you could hack it together. Streetview is a feature of the normal maps and you can load it with dragging the little person around. This library is only meant to handle basic Maps JavaScript API

https://developers.google.com/maps/documentation/javascript/streetview

**Comment by msreekm** [comment]
i dont want to click or drag, basically load the map with streetview.

**Comment by Qwiso** [comment]
There is an available method that can be your starting point. Within the component that you use, look closer at `this.props.google.maps.StreetViewPanorama`

https://developers.google.com/maps/documentation/javascript/examples/streetview-service

**Comment by andrehigher** [comment]
You just need to pass `streetViewControl` to the main component:

```
<Map
  ref='map'
  google={window.google}
  streetViewControl
  ...
 />
```
